### PR TITLE
Add monthly task for getting on-premises machines patched

### PR DESCRIPTION
**Pull request by MattGal**
### To double check:

* [x] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation (just a doc change)

### CODE REVIEW

**Comment by MattGal** [comment]
@Chrisboh  / @garath  / @jonfortescue FYI.

File: Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md [hunk 1]
```diff
@@ -132,6 +132,79 @@ If you feel this removal is in error, or believe a specific expiration should be
     - Create new issues in [dotnet/arcade](https://github.com/dotnet/arcade/) identifying the queue and linking to the build. Include links to these issues under the "queues that will expire in the next 7 days" or "after the next 7 days" headings as appropriate.
 - When all the above is complete for a given week, you may close the issue.
 
+##### Monthly (closest business day to the 15th):
+**Request DDFUN updates for on-premises Helix machines**
+
+We depend on our partner team DevDiv Fundamentals ("DDFUN") for machine maintenance tasks. DDFUN vendors use table "OsobInventory" in storage account 'helixscripts2' to view the machines.  We want any machine that shows up in HelixEnvironment "Production" with Partition Key not equal to "Not in Helix" to be at least reviewed for update monthly. See @MattGal or @Ilyas1974 if you believe you need to gain read access to this table. If new queues are found in the table that are not below, please make a pull request to update the list.
```
Comment (jonfortescue) [inline Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:138]: maybe make @MattGal & @ilyas1974 into links?

File: Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md [hunk 2]
```diff
@@ -132,6 +132,79 @@ If you feel this removal is in error, or believe a specific expiration should be
...
+
+We need to ensure and drive that the machines we are running are updated to the latest possible versions. Since we cannot rely on AzSecPack and automated reporting for these updates, we'll generate IcM tickets requesting this. For each of the three categories of operating system below (Linux, MacOS, and Windows) please create an IcM ticket [here](https://ddfunlandingpagev120210311150648.azurewebsites.net/) using the "[Other](https://portal.microsofticm.com/imp/v3/incidents/create?tmpl=E3619N)" template.
```
Comment (jonfortescue) [inline Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:140]: This should either specify which link you should click once you go to the ddfunlandingpage or be a deep link to the specific page needed.

File: Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md [hunk 3]
```diff
@@ -132,6 +132,79 @@ If you feel this removal is in error, or believe a specific expiration should be
...
+
+After creating such an IcM for each of the three groups of machines, use [this template](https://dnceng.visualstudio.com/internal/_workitems/create/Task?templateId=5ad0c1bc-5e95-45a1-b40b-de81c12b5b4a&ownerId=3f024d6c-9884-4f38-a598-025fca9dfcd2) to link the three IcM tickets, and keep the issue open until all three are resolved.
+
+
+Suggested IcM Description:
+```
+Machines in the following queues need to be updated to their latest patch versions.
+
+<List of queues from below>
```
Comment (jonfortescue) [inline Documentation/Operations/Helix-Machine-Management/Helix-Machine-Lifecycle-Processes.md:149]: Dunno if you should put this in angle brackets as someone unfamiliar with markup notation might replicate them.

**Review by jonfortescue — APPROVED** [review]

**Pull request merged**